packages/react-native-reanimated/src/specs/ReanimatedViewProvider.native.ts

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+import type { HostComponent } from 'react-native';
+
+import { shouldBeUseWeb } from '../PlatformChecker';
+
+let ReanimatedNativeView: HostComponent<any>;
+if (shouldBeUseWeb()) {
+  // this is a workaround for the jest
+  ReanimatedNativeView = {} as HostComponent<any>;
```

Comment:
1. Shouldn't we use `isJest()` instead?
2. Does react-native-gesture-handler need to do the same thing? Maybe we should add the workaround for Jest at a different abstraction layer?

Reply:
1. Yes, we can 👍
2. Jest is awful - but I have come up with another idea